core-web/libs/portlets/edit-ema/portlet/src/lib/dot-ema-shell/dot-ema-shell.component.ts

```diff
@@ -107,7 +107,7 @@ export class DotEmaShellComponent implements OnInit, OnDestroy {
 
             if (
                 isLayoutDisabled &&
-                this.activatedRoute.snapshot?.firstChild?.routeConfig.path === 'layout'
+                this.activatedRoute.firstChild.snapshot.url[0].path === 'layout'
```

Comment:
I recently change this line
Both lines do the "same thing".
But the previous line was much more complicated to test.
Furthermore, it is more advisable to obtain the current path through firstChild, since it directly has the data we need (The children of this route in the router state tree)
than by the snapshot.firsChild.routeConfig. This data is very nested